.gitignore

```diff
@@ -1,2 +1,5 @@
 .vagrant/
+.bundle/
 Vagrantfile
+vendor/bundle/
+Gemfile.lock
```

Comment:
philosophy question: if you want to force all your peers to have exactly the same gemset as your are using, than one can check it in. but if you have windows, linux, mac, bsd people it can be complicated to use the same gemset for every one.

i often like to check it in, but also often see it in the .gitignore. i put it in here, because it was not part of the repo in before and i didn't wanted to introduce a new file.

Reply:
This does not affect the container, but the git repo itself. In this case it is OK to have the lock file ignored.